liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/ServerFeatureSupport.java

```diff
@@ -191,6 +213,77 @@ private static boolean isSubModule(MavenProject potentialTopModule, MavenProject
         }
         return false;
     }
+    
+    protected List<MavenProject> getRelevantMultiModuleProjects(ProjectDependencyGraph graph) throws MojoExecutionException {
+    	getLog().debug("Resolve projects");
+    	
+        List<MavenProject> sortedReactorProjects = graph.getSortedProjects();
+        Set<MavenProject> conflicts = new LinkedHashSet<MavenProject>(); // keeps the order of items added in
+
+        // A leaf here is a module without any downstream modules depending on it
+        List<MavenProject> leaves = new ArrayList<MavenProject>();
+        for (MavenProject reactorProject : sortedReactorProjects) {
+            if (graph.getDownstreamProjects(reactorProject, true).isEmpty()) {
+            	getLog().debug("Found final downstream project: " + reactorProject.getArtifactId());
+            	
+            	if (skipConfigured(reactorProject)) {
+            		getLog().debug("Skip configured - skipping");
+            	} else if (reactorProject.getPackaging().equals("jar")) {
+            		getLog().debug("Jar project type - skipping");
+            	} else {
+                    leaves.add(reactorProject);
+            	}
+            }
+        }
+            
+        // Find any remaining conflicts
+        for (MavenProject leaf1 : leaves) {
+            for (MavenProject leaf2 : leaves) {
+                if (leaf1 != leaf2 && !(isSubModule(leaf2, leaf1) || isSubModule(leaf1, leaf2)) ) {
```

Comment:
Can you add a comment to explain what this conflict checking is handling? It seems like it is adding any leaf to `conflicts` that are not somehow related to each other in a module/submodule relationship. Did I interpret that correctly?

Reply:
Yup that's right - comments added.